src/errors.rs

```diff
@@ -84,7 +80,7 @@ impl<E> From<WireError<E>> for Error<E> {
 #[derive(Deserialize, Serialize, Clone, Debug)]
 pub enum WireError<E> {
     /// Error in serializing the server response or deserializing the client request.
```

Comment:
Is this comment still accurate? If so it's confusing that this can be for a failed deserialize of a response.